### DISCUSSION TITLE
Marking of axis not starting at 0

### DISCUSSION THREAD

**Discussion by MoBosse** [open-sdg/open-sdg#1402] [Ideas]
Is it possible to somehow mark it when an axis is broken / does not start at the value 0?

![grafik](https://user-images.githubusercontent.com/50657629/65597555-b6b7cd00-df99-11e9-9a37-911ea3838e1f.png)

**Comment by LucyGwilliamAdmin** [comment]
Just found [this](https://canvasjs.com/javascript-charts/chart-axis-scale-break/)

**Comment by MoBosse** [comment]
@LucyGwilliamAdmin: Thanks for the research. Do you have any idea of how to implement that?
 
I´ve tried to add
`axisY: {
        scaleBreaks: {
          customBreaks: [{
            startValue: 0,
            endValue: 2,
            color: "orange"
          }]
        }
      },`

in chartConfig in indicatorView.js but it did not work.

**Comment by LucyGwilliamAdmin** [comment]
@MoBosse I've just realised that the link I sent is for canvas js rather than chart js.

@brockfanning is it possible to use canvas js?

**Comment by brockfanning** [comment]
All, sorry I missed this. I think we should first look to see if it's possible in Chart.js. If it's not already possible in Chart.js, it may actually be easier to implement it (via a "plugin" or a patch for Chart.js) than to switch Open SDG to a different charting library.

**Comment by LucyGwilliamAdmin** [comment]
Ah right, wasn't sure if possible to use canvas.js things in chart.js but I think from your response, that's a no?

**Comment by brockfanning** [comment]
Right - our platform is pretty tightly coupled with Chart.js at this point. Canvas.js seems to be an alternative to Chart.js. In the long run it would be great to be flexible about the chart library that is used, but at the moment we're relying on Chart.js-specific code.

**Comment by brockfanning** [comment]
If this is not possible in Chart.js let's post an issue there for the feature request.

**Comment by LucyGwilliamAdmin** [comment]
Actually seems to be an issue open already re. this:

https://github.com/chartjs/Chart.js/issues/6737

Doesn't seem to really be any activity on it though

**Comment by LucyGwilliamAdmin** [comment]
We would need Chart.js to implement this before we can do anything so locking discussion
